content/en/docs/installation/kubernetes.md

```diff
@@ -106,9 +113,17 @@ In order to install the Helm chart, you must follow these steps.
 
 Install the `CustomResourceDefinition` resources separately.
 ```bash
-$ kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.13.1/deploy/manifests/00-crds.yaml
+# Kubernetes 1.15+
+$ kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.14.0/deploy/manifests/00-crds.yaml
+
+# Kubernetes <1.15
+$ kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.14.0/deploy/manifests/cert-manager-legacy.crds.yaml
```

Comment:
This file isn't being made available/does not exist anywhere. Perhaps we should include `crds-only.yaml` and `crds-only-legacy.yaml` as attachments to the *GitHub release*? Over time, we could then remove the `00-crds.yaml` file from the repository altogether.

Reply:
Sound like a good idea!
Wrote something up for this here: https://github.com/jetstack/cert-manager/issues/2660

Reply:
We should update both of these links to reference the versions published as part of the *GitHub release* now - we can then gradually begin phasing out the `00-crds.yaml` file in the repository altogether 😄

Reply:
Ping ^ the currently linked file doesn't exist - can you link to the assets attached to the release (in both of these examples + the openshift page)?